Code/Framework/AzFramework/AzFramework/Spawnable/SpawnableSystemComponent.h

```diff
@@ -77,6 +79,16 @@ namespace AzFramework
         void OnRootSpawnableReady(AZ::Data::Asset<Spawnable> rootSpawnable, uint32_t generation) override;
         void OnRootSpawnableReleased(uint32_t generation) override;
 
+        //
+        // SpawnableNotifications
+        //
+
+        //! Sent any time a SpawnEntityTicket is created through script
+        void OnSpawnEntityTicketCreated(EntitySpawnTicket spawnTicket) override;
+
+        //! Sent when the ScriptMediator system shuts down
+        void ClearEntityTickets() override;
```

Comment:
Should this also be triggered when loading a different level? I don't think spawned objects are meant to stay in the scene by default.